CHANGELOG.md

```diff
@@ -2,7 +2,15 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
-## [v7.1.1](https://github.com/puppetlabs/puppetlabs-java/tree/v7.1.1) (2021-08-25)
+## [v7.1.3](https://github.com/puppetlabs/puppetlabs-java/tree/v7.1.3) (2021-09-06)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-java/compare/v7.1.1...v7.1.3)
```

Comment:
okay I'm confused. I started the github action based on master, and that created a second changelog entry for 7.1.1. that's clearly not what we want. so I created a local branch, bumped the version in metadata.json to 7.1.2 and started the action again and it bumped the version to 7.1.3?